bitnami/contour/values.yaml

```diff
@@ -690,7 +690,17 @@ envoy:
   ## @param envoy.containerSecurityContext.runAsUser User ID for the Envoy container (to change this, http and https containerPorts must be set to >1024)
   ## @param envoy.containerSecurityContext.runAsNonRoot Run as non root
   ##
-  containerSecurityContext:
+  containerSecurityContextEnvoy:
+    enabled: true
+    runAsUser: 1001
+    runAsNonRoot: true
+  ## Shutdown Manager container security context
+  containerSecurityContextShutdownManager:
+    enabled: true
+    runAsUser: 1001
+    runAsNonRoot: true
```

Comment:
I would move the shutdown manager related configuration like this

* keep envoy.containerSecurityContext as it is, it will contain the information regarding the envoy container
* Move the second block to envoy.shutdownManager.containerSecurityContext, it will contain the information regarding the shutdown manager container

Reply:
yes that make sense; and thus it won't break backward compatibility. I believe it is fixed now.